src/frontend/Player/createVideojsPlayer.ts

```diff
@@ -27,6 +34,32 @@ export const createVideojsPlayer = (
   // add the video-js class name to the video attribute.
   videoNode.classList.add('video-js', 'vjs-big-play-centered');
 
+  const sources: VideoJsExtendedSourceObject[] = [];
+  const plugins: VideoJsPlayerPluginOptions = {};
+
+  if (!isMSESupported()) {
+    plugins.qualitySelector = {
+      default: '480',
+    };
+    const resolutions = Object.keys(video.urls.mp4).map(
+      (size) => Number(size) as videoSize,
+    );
+    resolutions
+      .sort((a, b) => b - a)
+      .forEach((size) => {
+        sources.push({
+          type: 'video/mp4',
+          src: video.urls.mp4[Number(size) as videoSize]!,
+          size: `${size}`,
+        });
+      });
```

Comment:
I do not think a variable is required here. You could also remove the `map` loop but you have to cast value in `sort` function and also in `forEach`.
```suggestion
    // Add mp4 sources by resolutions sorted by desc
    Object.keys(video.urls.mp4)
      .map(size => Number(size) as videoSize);
      .sort((a, b) => b - a)
      .forEach((size) => {
        sources.push({
          type: 'video/mp4',
          src: video.urls.mp4[size]!,
          size: size.toString(),
        });
      });
```